go/pkg/framework/server.go

```diff
@@ -64,6 +89,11 @@ func (s *AbstractMlrunGRPCServer) getServerOpts() []grpc.ServerOption {
 	return s.grpcServerOpts
 }
 
+func (s *AbstractMlrunGRPCServer) recoverFromPanic(ctx context.Context, p interface{}) error {
```

Comment:
```suggestion
func (s *AbstractMlrunGRPCServer) recoverFromPanic(ctx context.Context, panic interface{}) error {
```
Maybe also add log above the function on the function's purpose